include/z64message.h

```diff
@@ -8,10 +8,23 @@
 struct Actor;
 struct MessageTableEntry;
 struct OcarinaStaff;
+struct PlayState;
 
 extern u16 sBombersNotebookEventMessages[BOMBERS_NOTEBOOK_EVENT_MAX];
 extern u16 gBombersNotebookWeekEventFlags[BOMBERS_NOTEBOOK_EVENT_MAX];
 
+typedef enum FaceReactionSet {
+    /* 0x11 */ FACE_REACTION_SET_JIM = 0x11,
+    /* 0x12 */ FACE_REACTION_SET_BOMBERS,
+    /* 0x15 */ FACE_REACTION_SET_BOMBERS_HIDEOUT_GUARD = 0x15,
+    /* 0x2D */ FACE_REACTION_SET_TREASURE_CHEST_SHOP_GAL = 0x2D,
+    /* 0x2F */ FACE_REACTION_SET_CURIOSITY_SHOP_MAN = 0x2F,
+    /* 0x30 */ FACE_REACTION_SET_TOWN_SHOOTING_GALLERY_MAN,
+    /* 0x31 */ FACE_REACTION_SET_SWAMP_SHOOTING_GALLERY_MAN,
+    /* 0x36 */ FACE_REACTION_SET_PART_TIMER = 0x36
+} FaceReactionSet;
+
+u16 Text_GetFaceReaction(struct PlayState* play, FaceReactionSet reactionSet);
 typedef enum TextState {
```

Comment:
Sorry for being annoying, but can you add an empty line between those?

Reply:
You're not being annoying! I just missed this (or rather, my IDE was *already* showing a blank line here when none actually existed in practice).